src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24966.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if !MACCATALYST && !WINDOWS
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	internal class Issue24966 : _IssuesUITest
+	{
+		public Issue24966(TestDevice device) : base(device) { }
+
+		public override string Issue => "CollectionView, the footer moves to the bottom of the page when the empty view or empty view template is enabled";
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		public void CollectionViewFooterMovestoBottomWithEmptyvieworEmptyviewTemplate()
```

Comment:
Test is failing

Reply:
<img width="515" alt="image" src="https://github.com/user-attachments/assets/445d83ab-7cba-4f00-9a1c-6072f43df8e1">

Reply:
Since updating to .NET 9 and enabling `CollectionViewHandler2` in the iOS testing projects, we’re encountering UI test failures due to layout differences between `CollectionViewHandler`(with our custom fix in .NET 8) and `CollectionViewHandler2`. In the updated handler, the empty view is positioned incorrectly, overlapping the header, which alters the expected UI layout and causes screenshot comparison failures in tests.
The previous alignment fix for the empty view does not work with `CollectionViewHandler2` due to differences in rendering flow. We are currently working on fixing this issue to ensure similar behavior to `CollectionViewHandler` for consistent UI rendering across versions.

A separate issue has been created to track progress: https://github.com/dotnet/maui/issues/25606,